README.md

```diff
@@ -4,127 +4,132 @@
 
 ## Introduction
 
-This helm chart installs the [Scalyr Agent monitor](https://app.scalyr.com/help/scalyr-agent) into a Kubernetes
-cluster. It supports installing the agent with all features required to support a Kubernetes cluster monitoring.
-Additionally, it can deploy Scalyr agents which monitor other parts of the infrastructure (for example a hosted
-database service etc.).
+This helm chart installs the [Scalyr Agent](https://app.scalyr.com/help/scalyr-agent) in a Kubernetes
+cluster. Two Agent Plugins plugins are also installed:
+- The [Kubernetes monitor](https://app.scalyr.com/monitors/kubernetes) enables the Agent to monitor Kubernetes clusters.
```

Comment:
Kubernetes monitor is responsible for a couple of things:

1. Discovering pods / containers on the node and scheduling log collection for the matched pods (to simplify it, we could just say it's responsible for collecting pod logs)
2. Collecting metrics for the containers which run on that node